pkg/handler/webhook.go

```diff
@@ -206,3 +208,44 @@ func (h *Handler) handleGuildCreate(c *gin.Context, data json.RawMessage) {
 
 	c.JSON(http.StatusOK, gin.H{"status": "OK"})
 }
+
+func (h *Handler) handleMessageReactionAdd(c *gin.Context, data json.RawMessage) {
+	var req request.CreateMessageRepostHistRequest
+
+	byteData, err := data.MarshalJSON()
+	if err != nil {
+		c.JSON(http.StatusInternalServerError, gin.H{"error": err.Error()})
+		return
+	}
+
+	if err := discordgo.Unmarshal(byteData, &req); err != nil {
+		c.JSON(http.StatusBadRequest, gin.H{"error": err.Error()})
+		return
+	}
+
+	conf, err := h.entities.GetGuildRepostReactionConfigByReaction(req.GuildID, req.Reaction)
+	if err != nil {
+		c.JSON(http.StatusInternalServerError, gin.H{"error": err.Error()})
+		return
+	}
+	if req.ReactionCount < conf.Quantity {
```

Comment:
why do we return error here? we should ignore and return if it hasn't reached the required quantity to repost.
can put a log here to track if necessary